Soccar [UI]/src/nl/soccar/ui/fx/drawable/ScoreBoardUiFx.java

```diff
@@ -0,0 +1,63 @@
+package nl.soccar.ui.fx.drawable;
+
+import javafx.scene.canvas.GraphicsContext;
+import javafx.scene.text.TextAlignment;
+import nl.soccar.library.Game;
+import nl.soccar.library.enumeration.EventType;
+import nl.soccar.ui.DisplayConstants;
+import nl.soccar.ui.drawable.Drawable;
+import nl.soccar.ui.drawable.GameCanvas;
+import nl.soccar.ui.util.PhysicsUtilities;
+
+/**
+ * A ScoreBoardUiFx object represents a JavaFX Drawable of the Scoreboard.
+ * It keeps track of the Notification and NotificationPhysics models and provides methods to draw and
+ * update the models.
+ *
+ * @author PTS34A
+ */
+public class ScoreBoardUiFx extends Drawable<Game> {
+
+    /**
+     * Initiates a new NotificationUiFx Object using the given parameters.
+     *
+     * @param canvas The canvas on which this Notification is placed.
+     */
+    public ScoreBoardUiFx(GameCanvas canvas, Game game) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): game [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)